index.html

```diff
@@ -25,19 +28,23 @@
     <header class="profile">
       <nav class="profile-menu">
         <div class="logo roboto"><a href="#">Obote</a></div>
-        <ul class="menu-items">
-          <li><a href="#portfolio">Portfolio</a></li>
-          <li><a href="#about">About</a></li>
-          <li><a href="#contact">Contact</a></li>
+        <ul id="menu" class="menu-items">
+          <li class="list-item"><a href="#portfolio">Portfolio</a></li>
+          <li class="list-item"><a href="#about">About</a></li>
+          <li class="list-item"><a href="#contact">Contact</a></li>
         </ul>
         <img
           class="msg-icon"
           src="./images/Icons/Union.svg"
           alt="message icon"
         />
-        <button class="hamburger">
-          <img src="./images/Icons/menu-icon.png" alt="menu icon" />
+        <button id="toggleButton" class="hamburger">
+          <span class="fas fa-bars"></span>
         </button>
+        <button id="close" class="hamburger hide">
+          <span class="fas fa-times"></span>
+        </button>
+
```

Comment:
- [ ] In the mobile view the menu Icon is missing. Kindly fix the issue.
![image](https://user-images.githubusercontent.com/43698511/135930263-c955f354-5b74-4d6c-9e88-818e1530a811.png)

Reply:
hello miss we are using a dependency called fontawesome.

Reply:
please run npm install to view the icons